sys/docker/Dockerfile

```diff
@@ -31,7 +30,7 @@ RUN apt-get update \
     && cp /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini \
     && rm -rf /var/lib/apt/lists/*
 
-RUN wget https://raw.githubusercontent.com/composer/getcomposer.org/76a7060ccb93902cd7576b67264ad91c8a2700e2/web/installer -O - -q | php -- --quiet \
+RUN curl -sL https://raw.githubusercontent.com/composer/getcomposer.org/76a7060ccb93902cd7576b67264ad91c8a2700e2/web/installer | php -- --quiet \
```

Comment:
what do you think about override with `COPY --from=composer /usr/bin/composer /usr/bin/composer`?

Reply:
done